packages/hash/api/src/seed-data/seed-pages.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { PageModel } from "../model";
+
+export type PageDefinition = {
+  title: string;
+  nestedPages: PageDefinition[];
+};
```

Comment:
Nit: `nestedPages` could be undefined to reduce the size of the definitions somewhat
```suggestion
export type PageDefinition = {
  title: string;
  nestedPages?: PageDefinition[];
};
```

Reply:
I was explicit on purpose, but I suppose it doesn't hurt to have it be optional here :)

Reply:
Addressed in a4632932ca9205874fd7142e2d0a106d1e15d0f5